keps/sig-node/277-ephemeral-containers/kep.yaml

```diff
@@ -6,7 +6,7 @@ owning-sig: sig-node
 participating-sigs:
   - sig-auth
   - sig-node
-status: implementable
+status: implemented
```

Comment:
@kikisdeliveryservice @palnabarun do we set this now or after all PRs are merged?

Reply:
We should set the status to `implemented` after the release has happened where the feature graduates to Stable/GA. Currently, it should still be `implementable`.

Reply:
reverted back to implementable, thanks.